tests/python/contrib/test_clml/test_ops.py

```diff
@@ -54,11 +62,8 @@ def _get_conv_model(
         shape = (shape[0], shape[1], shape[2] + padding[0] * 2, shape[3] + padding[1] * 2)
     is_depthwise = shape[1] == channels == groups
 
-    weight_format = "OIHW" if is_depthwise else "OIHW"
-    if weight_format == "IOHW":
-        weight_shape = (shape[1] // groups, channels, kernel_h, kernel_w)
-    else:
-        weight_shape = (channels, shape[1] // groups, kernel_h, kernel_w)
+    weight_format = "OIHW"
```

Comment:
Why did you remove check on the dephtwise convolution?

Reply:
This was the issue we had earlier with depthwise.

Weight layout transform for depthwise is handled internally by CLML compilation process internally. https://github.com/apache/tvm/blob/6f6bf1912e08f5113c1ac438ea600270b479055d/python/tvm/relay/op/contrib/clml.py#L79